devicon.json

```diff
@@ -7921,6 +7921,33 @@
         "color": "#173aee",
         "aliases": []
     },
+    {
+        "name": "scikitlearn",
+        "altnames": [
+            "scikit-learn",
+            "scikit",
+            "sklearn"
+        ],
+        "tags": [
+            "machine-learning",
+            "python",
+            "tool",
+            "library"
+        ],
+        "versions": {
+            "svg": [
+                "original-wordmark",
+                "plain-wordmark",
+                "line-wordmark"
+            ],
+            "font": [
+                "plain-wordmark",
+                "line-wordmark"
+            ]
+        },
+        "color": "#f89939",
+        "aliases": []
```

Comment:
Now that I think about it: shouldn't we add aliases here?

Reply:
You're right, nice catch!

There are two alternatives 

1. `wordmark` should be removed from from the filename, and aliases should be added for wordmark... So like this:

base: original,
alias: original-wordmark

base: plain
alias: plain-wordmark


... Etc

2. Remove wordmark from the icon names, and add the wordmark versions I suggested in this comment: https://github.com/devicons/devicon/pull/1616#pullrequestreview-1241187222

Reply:
I followed the first option, renamed the files, and added aliases.